var/spack/repos/builtin/packages/mesa/package.py

```diff
@@ -147,6 +148,7 @@ class Mesa(MesonPackage):
     # on development versions of LLVM 13 may or may not have it. Use SFINAE to detect
     # the existence of the function and call it only if it is available.
     patch("handle_missing_set_override_stack_alignment.patch", when="@21.2.3:")
+    patch("disable-gallivm-coroutine-for-libllvm15.patch", when="@22.1.2:")
```

Comment:
If mesa constains way llvm 15 for this version, then I don't think it requires a patch. Unless there is a strong need for having a mesa build with 5.3.0 right now I would prefer not to patch things.

Reply:
I think all of the E4S deployments are using ROCm 5.2.0. @eugeneswalker @wspear could correct me if I missed an update recently.

Reply:
> I think all of the E4S deployments are using ROCm 5.2.0. @eugeneswalker @wspear could correct me if I missed an update recently.

All of the current ROCm-enabled site deployments (i.e. Crusher) use ROCm 5.2.0, yes.